Library/Homebrew/utils.rb

```diff
@@ -103,7 +103,7 @@ def odeprecated(method, replacement = nil, disable: false, disable_on: nil, call
      Homebrew.raise_deprecation_exceptions?
     raise MethodDeprecatedError, message
   else
```

Comment:
`elsif !Homebrew.auditing?`

Reply:
Doesn't this get rid of the default case? Currently, that should not be an issue, but it does add a single point of failure for the `odeprecated` code message printout.